pkg/controller/container-runtime-config/container_runtime_config_controller_test.go

```diff
@@ -188,6 +188,8 @@ func (f *fixture) newController() *Controller {
 		i.Machineconfiguration().V1().ControllerConfigs(),
 		i.Machineconfiguration().V1().ContainerRuntimeConfigs(),
 		ci.Config().V1().Images(),
+		ci.Config().V1().ImageDigestMirrorSets(),
+		ci.Config().V1().ImageTagMirrorSets(),
```

Comment:
Shouldn’t there be _some_ tests that the IDMS/ITMS data are correctly propagated? Around `TestRunImageBootstrap`/ some future variant of `TestICSPUpdate`?